src/Http/Controllers/OpenApiJsonController.php

```diff
@@ -13,7 +13,9 @@ public function __invoke()
     {
         $json = $this->getJson();
 
-        $json = $this->configureServer($json);
+        if (config('swagger-ui.local_server_only')) {
+            $json = $this->configureServer($json);
+        }
```

Comment:
can you move the if-statement inside the configureServer method

```php
protected function configureServer(array $json)
{
    if (! config('swagger-ui.use_app_url')) {
        return $json;
    }

    // ...
}
```

Reply:
Can you also add a test for this behavior?

Reply:
ok, did it